.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  pull_request:
+  schedule:
+    # Weekly Monday 9AM build
+    - cron: '0 9 * * 1'
+
+jobs:
+  tox:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Python 3.6
+            runs-on: ubuntu-latest
+            python-version: 3.6
+            toxenv: py36
```

Comment:
Should we drop Python 3.6? Its end of life is the end of December.
3.7?

Reply:
Instead add 3.10?

Reply:
I was thinking the same

Reply:
I'll update the package template as well